misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JMethod.java

```diff
@@ -168,8 +171,16 @@ private void setReturnTypeAttributes(Class returnTypeClass) {
                 isStringArrayReturn = true;
             } else {
                 returnComponentType = getAlias(returnTypeClass.getComponentType(), env.getAliases());
+                if (isOptionalReturnTypes) {
+                    returnComponentType = returnComponentType + "?";
```

Comment:
Shall we use a constant for "?" since it has been used in multiple places?

Reply:
Added